micrometer-core/src/main/java/io/micrometer/core/instrument/binder/mongodb/DefaultMongoMetricsCommandTagsProvider.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.mongodb;
+
+import com.mongodb.event.CommandEvent;
+import com.mongodb.event.CommandFailedEvent;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+
+/**
+ * Default implementation for {@link MongoMetricsCommandTagsProvider}.
+ *
+ * @author Chris Bono
+ */
+public class DefaultMongoMetricsCommandTagsProvider implements MongoMetricsCommandTagsProvider {
+
+    @Override
+    public Iterable<Tag> commandTags(CommandEvent event) {
+        return Tags.of(
+                Tag.of("command", event.getCommandName()),
+                Tag.of("cluster.id", event.getConnectionDescription().getConnectionId().getServerId().getClusterId().getValue()),
+                Tag.of("server.address", event.getConnectionDescription().getServerAddress().toString()),
+                Tag.of("status", (event instanceof CommandFailedEvent) ? "FAILED" : "SUCCESS"));
```

Comment:
What do you think about checking `CommandSucceededEvent` and fall back to `"FAILED"` instead?
So if for any reason a third type of event will be passed in the future, it will mark this as `"FAILED"` which will draw better user attention than `"SUCCESS"` so there is a higher chance that the bug will be reported.